rts/motoko-rts/src/types.rs

```diff
@@ -452,6 +453,22 @@ impl Array {
     }
 }
 
+#[rustfmt::skip]
+#[repr(C)] // See the note at the beginning of this module
+pub struct Region {
+    pub header: Obj,
+    pub id: u16,
+    pub page_count: u64,
```

Comment:
u32 is probably more than enough, realistically, for pagecoung, and you might want to add a u16 worth of padding (pub padding : u16) to make everything 32-bit, word-aligned